File: src/main/java/dev/fabby/com/commands/commandargs/NoArgTabCompleter.java
package dev.fabby.com.commands.commandargs;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;
import org.bukkit.command.TabCompleter;

import java.util.ArrayList;
import java.util.List;

public class NoArgTabCompleter implements TabCompleter {

    @Override
    public List<String> onTabComplete(CommandSender commandSender, Command command, String s, String[] args) {
        List<String> list = new ArrayList<>();
        list.add("");
        return list;
    }
}

File: src/main/java/dev/fabby/com/commands/commandargs/NoArgTabCompleter.kt
package dev.fabby.com.commands.commandargs

import org.bukkit.command.Command
import org.bukkit.command.CommandSender
import org.bukkit.command.TabCompleter

import java.util.ArrayList

class NoArgTabCompleter : TabCompleter {

    override fun onTabComplete(
        commandSender: CommandSender,
        command: Command,
        s: String,
        args: Array<String>
    ): List<String> {

        val list = ArrayList<String>()
        list.add("")
        return list
    }
}